### PR TITLE
builder/users: add networkexception

### PR DESCRIPTION
**Pull request by networkException**
- [x] I have read the entire README https://github.com/nix-community/aarch64-build-box
- [x] I completely understood the README https://github.com/nix-community/aarch64-build-box
- [x] I know when I can't trust the builder, as explained in the README

Hi!

I'm maintaining a few packages in nixpkgs, notably {ungoogled-,}chromium. This requires somewhat frequently rebuilding chromium entirely, for which I currently use my desktop at home as a remote builder (not too great to block the machine for 90+ minutes).
I'm also looking into maintaining more chromium related packages, such as re2 ([which requires rebuilding quite a bit of the world](https://github.com/NixOS/nixpkgs/pull/249560)).

It would be lovely to have some dedicated build infrastructure for working on nixpkgs :)

### CODE REVIEW

**Comment by zowoq** [comment]
We don't really have specific rules for allowing access to this machine but it is basically limited to people who are well known in the nixpkgs/nixos/etc community, is there somebody who is known who can vouch for you?

**Comment by emilylange** [comment]
Can I -- currently nixpkgs committer -- vouch for @networkException? :)

File: modules/nixos/community-builder/users.nix [hunk 1]
```diff
@@ -54,6 +54,11 @@ let
       keys = ./keys/lily;
     };
 
+    networkexception = {
+      trusted = true;
+      keys = ./keys/networkexception;
+    };
```
Comment (zowoq) [inline modules/nixos/community-builder/users.nix:60]:
```suggestion
    networkexception.keys = ./keys/networkexception;
```

We've removed trusted from all users in https://github.com/nix-community/infra/pull/735.

**Review by zowoq — COMMENTED** [review]
Please rebase to fix the merge conflict.

**Comment by Mic92** [comment]
Applied in https://github.com/nix-community/infra/pull/798/files

**Pull request closed without merging**